.github/workflows/bundle-image-build-push.yml

```diff
@@ -0,0 +1,29 @@
+name: 'Build and Push Bundle Image'
```

Comment:
I think we can reduce the total number of workflows we have in the end if we were to name this one `.github/workflows/publish.yml`, wdyt?

The workflow.name is important, maybe "Publish Operator" (wouldn't be surprised if after making a release this workflow runs to build the images and push to community operators | helm charts.